src/CSSBattle/95_Pokeball/Yongbin.html

```diff
@@ -0,0 +1,43 @@
+<div class="circle">
+  <div class="gap"></div>
+  <div class="redgap"></div>
+  <div class="smallcircle"></div>
```

Comment:
Consider using camelCase or PascalCase to name your variables

Reply:
Resolved in commit 5ec733b.